docs/src/community/communication.md

```
@@ -26,10 +26,10 @@ There are two mailing lists used by Lance:
 
 ## Community Sync
 
-We hold Lance community syncs every other Thursday at 9am PT. Everyone is welcome to attend! 
+We hold Lance community syncs every other Thursday at 9am PT. Everyone is welcome to attend!
 
 - Subscribe to the [Lance Format mailing list](https://groups.google.com/a/lance.org/g/dev) to receive the meeting invite
 - Feel free to suggest discussion topics on the [meeting notes](https://docs.google.com/document/d/1cP058pJLVjj39DGaFV5yHI6qyWBwp1B2YKyIwXzYlAI/edit?tab=t.0)
 - Past recordings can be found [here](https://www.youtube.com/playlist?list=PLQysAafL04jUgOj7j6mCE5fcZEBB9Amu9)
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)